service/collector/env.go

```diff
@@ -26,11 +39,15 @@ var (
 type EnvConfig struct {
 	G8sClient versioned.Interface
 	Logger    micrologger.Logger
+
+	Directory string
```

Comment:
Why do we need to have that configurable?

Reply:
Don't know. We maybe just don't.

Reply:
Just going with a constant now.